components/kyma-environment-broker/internal/process/provisioning/apply_kyma_step_test.go

```diff
@@ -81,13 +105,13 @@ func TestCreatingKymaResource_UseNamespaceFromTimeOfCreationNotTemplate(t *testi
 
 	cli.List(context.Background(), &aList)
 	assert.Equal(t, 1, len(aList.Items))
-	assertLabelsExists(t, aList.Items[0])
+	assertLabelsExistsForInternalKymaResource(t, aList.Items[0])
 
 	svc.Run(operation, logrus.New())
 	assert.Equal(t, "namespace-in-time-of-creation", operation.KymaResourceNamespace)
 }
 
-func TestUpdatingKymaResourceIfExists(t *testing.T) {
+func TestUpdatinInternalKymaResourceIfExists(t *testing.T) {
```

Comment:
Is there the negative test? Testing if Kyma resource is not marked as internal.

Reply:
No. I can add.